assets/js/atomic/blocks/product-elements/price/block.tsx

```diff
@@ -53,8 +61,8 @@ export const Block = ( props ) => {
 		);
 	}
 
-	const prices = product.prices;
-	const currency = getCurrencyFromPriceResponse( prices );
+	const prices: PriceProps = product.prices;
+	const currency = getCurrencyFromPriceResponse( { ...prices } );
```

Comment:
This part had been changed to prevent a TS error after adding https://github.com/woocommerce/woocommerce-blocks/blob/31da04f30f10d14f5895167311bf460c04a79ecf/assets/js/atomic/blocks/product-elements/price/block.tsx#L22-L34.